data/json/items/tool/workshop.json

```diff
@@ -453,12 +453,43 @@
     "copy-from": "hammer",
     "ascii_picture": "hammer",
     "weight": "500 g",
+    "price": "17 USD",
```

Comment:
I specifically didn't do that. Nobody in their right mind would buy a chunk of copper on a stick for 17$. If this causes weird interaction ingame make it like 1$ or 2$ max, unless you've got links for ones for sale online